c7n/resources/s3.py

```diff
@@ -1160,12 +1160,31 @@ def process(self, resources):
 
         for r in resources:
             client = bucket_client(local_session(self.manager.session_factory), r)
-            target_prefix = self.data.get('target_prefix', r['Name'] + '/')
             if 'TargetBucket' in r['Logging']:
                 r['Logging'] = {'Status': 'Enabled'}
             else:
                 r['Logging'] = {'Status': 'Disabled'}
             if enabled and (r['Logging']['Status'] == 'Disabled'):
+                # code to support expand_variables()
+                session = local_session(self.manager.session_factory)
+                iam_client = session.client('iam')
+                aliases = iam_client.list_account_aliases().get('AccountAliases', ('',))
```

Comment:
can we drop the aliases as a variable, this is doing a separate iam call per resource, for the same constant value.